### PR TITLE
Move E2Es into UI repo

### PR DESCRIPTION
**Pull request by richpjames**
Following from CAS2 teams approach: https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/482 

It’s important to note that e2e tests are required to pass for deploying the frontend to preprod and prod.

We found that it can be difficult to coordinate the frontend changes (a pull request in 1 repo) alongside the changes to the e2e tests (a pull request in another repo).

Both must be worked on and tested locally from different repos. We also have double the admin of proposing and reverting two pull requests, rather than just one.

We’d like to move these tests from a separate repo to the same repo.
Changes

### Changes

 - We install playwright and copy over the e2e tests from the separate repo.

### CODE REVIEW

File: playwright-report/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+
+<!DOCTYPE html>
```
Comment (pezholio) [inline playwright-report/index.html:3]: This should probably be `.gitignore`d

File: playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import 'dotenv/config'
```
Comment (pezholio) [inline playwright.config.ts:1]: Can this sit in the `e2e` dir?

File: server/@types/shared/models/ApprovedPremisesApplication.ts [hunk 3]
```diff
@@ -2,31 +2,29 @@
...
-    status: ApprovedPremisesApplicationStatus;
-    assessmentId?: string;
-    assessmentDecision?: AssessmentDecision;
-    assessmentDecisionDate?: string;
-    submittedAt?: string;
-    personStatusOnSubmission?: PersonStatus;
-    apArea?: ApArea;
-});
-
+import type { AnyValue } from './AnyValue'
```
Comment (pezholio) [inline server/@types/shared/models/ApprovedPremisesApplication.ts:5]: How come this file has changed?

**Review by pezholio — APPROVED** [review]
LGTM - I think this is deffo worth doing. Just one thing on a folder that I don't think needs committing.

We'll also need to make sure the API is updated too once this is live

File: e2e/playwright/.auth/user.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
```
[the remark this answers is not in the document]
Comment (richpjames) [reply e2e/playwright/.auth/user.json:1]: oo yeah

**Comment by tahb** [comment]
[Recently discovered that I forgot to add additional folder ignores for tsconf](https://mojdt.slack.com/archives/C05GEPLBRN3/p1714121815149429?thread_ts=1714121651.920579&cid=C05GEPLBRN3) which you may also need.

**Comment by richpjames** [comment]
@tahb good shout, thanks

**Pull request merged**